src/encoded/schemas/sc_atac_multiplet_quality_metric.json

```diff
@@ -110,6 +159,94 @@
                     "title": "MD5sum"
                 }
             }
+        },
+        "barcode_pairs_expanded": {
+            "title": "Total barcode pair statistics",
+            "type": "object",
+            "additionalProperties": false,
+            "attachment": true,
+            "formInput": "file",
+            "properties": {
+                "download": {
+                    "title": "File Name",
+                    "type": "string"
+                },
+                "href": {
+                    "comment": "Internal webapp URL for document file",
+                    "type": "string"
+                },
+                "type": {
+                    "title": "MIME type",
+                    "type": "string",
+                    "enum": [
+                        "application/gzip"
```

Comment:
I don't think this a supported MIME type from: https://github.com/ENCODE-DCC/encoded/blob/dev/src/encoded/schemas/generic_quality_metric.json#L47-L58